modules/post/windows/gather/enum_patches.rb

```diff
@@ -34,32 +34,33 @@ def initialize(info={})
 
   def run
     unless load_extapi
-      print_error 'ExtAPI failed to load'
+      print_error("ExtAPI failed to load")
       return
     end
 
     begin
-      objects = session.extapi.wmi.query("SELECT HotFixID FROM Win32_QuickFixEngineering")
+      objects = session.extapi.wmi.query("SELECT HotFixID, InstalledOn FROM Win32_QuickFixEngineering")
     rescue RuntimeError
-      print_error "Known bug in WMI query, try migrating to another process"
+      print_error("Known bug in WMI query, try migrating to another process")
```

Comment:
See comment above for why one doesn't really need to change this as `(` and `)` are explicitly included when you call a function with only one parameter.

Reply:
I've changed this - I removed the parentheses and changed it to a single quote (per your guidelines above).